apps/camera_pipe/process.cpp

```diff
@@ -105,6 +105,9 @@ int main(int argc, char **argv) {
         fprintf(stderr, "%4d ", _lut[i]);
     }
     fprintf(stderr, "\n");
+    Image_info(lut);
```

Comment:
I think that this change (the image deubgging stuff) should be made as a separate PR to master (not the hexagon branch). It's independent of hexagon, and we'd like to suggest changes on it separately.